dot/types/tx.go

```diff
@@ -33,10 +33,24 @@ const (
 	TxnExternal
 )
 
-// TransactionPaymentQueryInfo represents the basic information of a given encoded extrinsic
-type TransactionPaymentQueryInfo struct {
+// RuntimeDispatchInfo represents information related to a dispatchable's class, weight, and fee that can be queried
+//  from the runtime
+type RuntimeDispatchInfo struct {
```

Comment:
Note: this was changed to `RuntimeDispatchInfo` to conform with naming used by paritytech: https://paritytech.github.io/substrate/master/pallet_transaction_payment_rpc_runtime_api/struct.RuntimeDispatchInfo.html#